Makefile

```diff
@@ -19,4 +20,4 @@ all: wit-deps-update wit-bindgen build
 # For testing components using the library locally with module replace.
 # In the published version binding/binding.c is git ignored, but locally we have to delete it.
 prepare-for-local-replace:
-	rm binding/binding.c
\ No newline at end of file
+	rm binding/binding.c
```

Comment:
is this still a thing that is generated?

Reply:
no idea, did not touch this

Reply:
lemme try it